packages/contracts/test/multi_sig_with_time_lock.ts

```diff
@@ -139,8 +139,7 @@ describe('MultiSigWalletWithTimeLock', () => {
                 expect(initialSecondsTimeLocked).to.be.equal(0);
 
                 txHash = await multiSig.executeTransaction.sendTransactionAsync(txId, { from: owners[0] });
-                await web3Wrapper.awaitTransactionMinedAsync(txHash, constants.AWAIT_TRANSACTION_MINED_MS);
```

Comment:
I think both web3Wrapper.awaitTransactionMinedAsync and zeroEx.awaitTransactionMinedAsync do the same thing, so I removed one of them. @fabioberger can you confirm that's correct?